Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -604,6 +607,36 @@ codeunit 4037 "Helper Functions"
         end;
     end;
 
+    local procedure CreateNoSeries()
+    begin
+        CreateNoSeriesForStandardCode('GP-SSC', 'Standard Sales Code', 'SC00000001');
```

Comment:
Can we move these to labels and possible to the single place so it is easy to find where it is used?